Multisig/UI/WalletConnect/WalletConnectServerController.swift

```diff
@@ -122,11 +122,11 @@ extension WalletConnectServerController: ServerDelegate {
                                             icons: [URL(string: "https://gnosis-safe.io/app/favicon.ico")!],
                                             url: URL(string: "https://gnosis-safe.io")!)
 
-        guard let safe = try? Safe.getSelected(), let address = safe.address else {
+        guard let safe = try? Safe.getSelected(), let address = safe.address, let network = safe.network else {
             let walletInfo = Session.WalletInfo(
                 approved: false,
                 accounts: [],
-                chainId: App.configuration.app.network.chainId,
+                chainId: 1,
```

Comment:
we should have these constants defined somewhere in one place, please replace this with a named constant